root/etc/services.d/jenkins/run

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/with-contenv sh
+
+# Start Jenkins, but remove the env var for the admin password to prevent it from leaking to a job
+exec env -u JENKINS_ADMIN_PASS /usr/local/bin/jenkins.sh
```

Comment:
You only need the ENV var is you want a specific admin password, otherwise it is blank.  I think for CI servers we willset this and for local leaving it blank is fine.